### ISSUE TITLE
Is this code consists of MS Graph/TODO API authorization?

### ISSUE THREAD

**Issue by mediaexplorer74** [TMinh25/ToDo#2]
Hi!

Very interesting project.
Please tell me , is your code consists of MS Graph authorization realization?

I want to port this proto over to UWP that is good for my old sweet phone Lumia 950 =)

**Comment by TMinh25** [comment]
> Hi!
> 
> Very interesting project. Please tell me , is your code consists of MS Graph authorization realization?
> 
> I want to port this proto over to UWP that is good for my old sweet phone Lumia 950 =)

I don't think so, this project I built with any knowledge about any structure in microsoft. It just simple SQL CRUD project for school.
